changelogs/fragments/517-add-ports-no-refresh-in-exos-facts.yaml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - exos_facts return timeout error when we use connection type network_cli.
+    It happened because we send command without 'no-refresh' and script cli2json.py stuck in loop while reading console output
```

Comment:
```suggestion
  - exos_facts - returns timeout error when we use connection type ``network_cli``.
    It happened because we send command without ``no-refresh`` and script ``cli2json.py`` stuck in loop while reading console output (https://github.com/ansible-collections/community.network/pull/517).
```

Reply:
I fixed it. Thank you

Reply:
cool, thanks, so let's wait until the mid/end of the next week, then I'll merge if no objections

Reply:
Ok, i hope that all fine. 
Thank you :)